README.md

```diff
@@ -56,88 +56,147 @@ Travis CI testing.
+```shell
+kubectl label nodes --all openwhisk-role=invoker
+```
+If you have a multi-node cluster, for each node <INVOKER_NODE_NAME>
+you want to be an invoker, execute
+```shell
+$ kubectl label nodes <INVOKER_NODE_NAME> openwhisk-role=invoker
+```
 
-## Initial Cluster Configuration
+## Customize the Deployment
 
-* Follow the steps for initial [Cluster Setup](kubernetes/cluster-setup)
-* Configure your [Ingresses](kubernetes/ingress), including configuring the wsk CLI.
+You will need to create a mycluster.yaml file that records how the
+OpenWhisk deployment on your cluster will be accessed by clients.  See
+the [ingress discussion](./docs/ingress.md) for details. Below is a sample
+file appropriate for a Minikube cluster where `minikube ip` returns
+`192.168.99.100` and port 31001 is available to be used.
 
-## Configure or Deploy CouchDB
+```yaml
+whisk:
+  ingress:
+    type: NodePort
+    api_host_name: 192.168.99.100
+    api_host_port: 31001
 
-Do one of the following:
-* For development and testing purposes, this repo includes a configuration
-  for deploying a [non-replicated CouchDB instance](kubernetes/couchdb)
-  within the Kubernetes cluster.
-* For a production level CouchDB instance, take a look at the main
-  OpenWhisk [documentation for configuring CouchDB](https://github.com/apache/incubator-openwhisk/blob/master/tools/db/README.md).
-  You will need to define the db.auth secret and db.config configmap as described in the [CouchDB README.md](kubernetes/couchdb/README.md)
-  to match your database deployment and create a CouchDB service instance
-  that forwards connections to your external database.
+nginx:
+  httpsNodePort: 31001
+```
 
-## Deploy Remaining Components
+Beyond specifying the ingress, the `mycluster.yaml` file is also used
+to customize your OpenWhisk deployment by enabling optional features
+and controlling the replication factor of the various micro-services
+that make up the OpenWhisk implementation. See the [configuration
+choices documentation](./docs/configurationChoices.md) for a
+discussion of the primary options.
 
-To deploy OpenWhisk on Kubernetes, you must deploy its components in
-an order that respects their dependencies.  Detailed instructions and
-the supporting configuration files can be found in the kubernetes
-directory tree. Follow the instructions for each step in order.
+## Deploy With Helm
 
-* Deploy [ApiGateway](kubernetes/apigateway)
-* Deploy [Zookeeper](kubernetes/zookeeper)
-* Deploy [Kafka](kubernetes/kafka)
-* Deploy [Controller](kubernetes/controller)
-* Deploy [Invoker](kubernetes/invoker)
-* Deploy [Nginx](kubernetes/nginx)
+Deployment can be done by using the following single command:
+```shell
+helm install . --namespace=openwhisk -f mycluster.yaml
+```
 
-## Install system actions and the openwhisk catalog
+After a while, if you can see all the pods listed by the command below
+are in `Running` state, congratulations, you have finished OpenWhisk
+deployment:
+```shell
```

Comment:
I think is better to suggest for user to use `helm status` first, and then as secondary to use `kubectl`

Reply:
done.